### PR TITLE
Add possibility to put false into config.footprints.controllers

### PR DESCRIPTION
**Pull request by jaumard**
Fix https://github.com/trailsjs/trailpack-footprints/issues/3

### CODE REVIEW

**Comment by jaumard** [comment]
@tjwebb what do you think of this ? Can be merge ?

**Comment by tjwebb** [comment]
Yes, can we also include docs + test?

**Comment by jaumard** [comment]
Of course, I will try to work on this this weekend and let you know :)

**Comment by jaumard** [comment]
Here it is, let me know if it's enough or not.

**Comment by jaumard** [comment]
Also fix https://github.com/trailsjs/trailpack-footprints/issues/8, I edit tests unit to include the prefix. 
Like this tests will crash if there a problem with it now.

**Comment by jaumard** [comment]
About https://github.com/trailsjs/trailpack/issues/31

**Comment by jaumard** [comment]
@tjwebb little up :) can I merge this ?

File: package.json [hunk 1]
```diff
@@ -12,10 +12,13 @@
   "keywords": [
     "trailpack",
     "trails",
-    "trailjs"
+    "trailjs",
+    "footprints",
+    "rest",
+    "restfull"
```
Comment (tjwebb) [inline package.json]: It's spelled "restful", with one L. English is weird :)

Comment (jaumard) [inline package.json]: Less than French but yeah sometime it's hard lol, sorry about this it's fixed now :)

**Pull request merged**